packages/main/src/plugin/kubernetes-context-state.ts

```diff
@@ -403,9 +403,9 @@ export class ContextsManager {
     );
   }
 
-  // update is the reconcile function, it gets as input the last known kube config
-  // and starts/stops informers for different kube contexts, depending on these inputs
   async update(kubeconfig: KubeConfig): Promise<void> {
+    const checkOnlyCurrentContext = kubeconfig.contexts.length > 10;
```

Comment:
IMHO the `10` should be a const at the top  of this file so we can change this in the future if it's an issue?